scripts/typescript_checks.py

```diff
@@ -74,7 +77,9 @@ def compile_and_check_typescript(config_path):
     if os.path.exists(COMPILED_JS_DIR):
         shutil.rmtree(COMPILED_JS_DIR)
 
-    error_messages = list(iter(process.stdout.readline, ''))
+    if process.stdout is not None:
```

Comment:
`process.stdout` will never be `None` because we specify that stdout should be captured in the `subprocess.Popen` call. See the Python docs: https://docs.python.org/3.8/library/subprocess.html#subprocess.CompletedProcess

Reply:
The assert sounds like a good solution to me!